pom.xml

```diff
@@ -6,7 +6,7 @@
   <artifactId>mondemand-java</artifactId>
   <packaging>jar</packaging>
   <name>mondemand-java</name>
-  <version>5.1.0-SNAPSHOT</version>
+  <version>6.0.0</version>
```

Comment:
major bump?  I think these changes are backwards compatible, since you're only adding a few new methods.  could be a minor bump instead.